Orange/widgets/visualize/owdistributions.py

```diff
@@ -214,6 +214,7 @@ def pdf(x, a, sigma=1, weights=None):
 
 class OWDistributions(OWWidget):
     name = "Distributions"
+    short_name = "Distribute"
```

Comment:
I don't like this change to a verb. Predictions -> Predict is OK, because the widget predicts and then shows predictions. But Distributions widget does not distribute anything. "Histogram" would be a better name anyway. Rename the whole widget? @BlazZupan?